themes/bootstrap5/templates/ajax/itemLocationList.phtml

```diff
@@ -0,0 +1,12 @@
+<?php $availabilityStatus = $this->availabilityStatus(); ?>
+<?php foreach ($this->locationList as $location): ?>
+  <div class="groupLocation">
+    <span class="<?= $availabilityStatus->getClass($location['availability'])?>">
+      <?=$this->icon($availabilityStatus->getIcon($location['availability']))?>
+      <?=$this->escapeHtml($location['location'])?>
+    </span>
+  </div>
+  <div class="groupCallnumber">
+    <?=$location['callnumber']?>
```

Comment:
This needs to be escaped.

Reply:
This is already rendered HTML from `itemCallnumber.phtml`, see `GetItemStatuses.php:374` and `GetItemStatuses.php:238-256`. I changed the name for less confusion in the class, check_item_statuses.js and template.